### PR TITLE
Add inventory items landing page [SCI-9506]

### PR DESCRIPTION
**Pull request by sboursen-scinote**
Jira ticket: [SCI-9506](https://scinote.atlassian.net/browse/SCI-9506)

### What was done
Add inventory items landing page

[SCI-9506]: https://scinote.atlassian.net/browse/SCI-9506?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/javascript/packs/vue/repository_item_error_sidebar.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/* global */
+
+import PerfectScrollbar from 'vue3-perfect-scrollbar';
+import { createApp } from 'vue/dist/vue.esm-bundler.js';
+import { vOnClickOutside } from '@vueuse/components';
```
Comment (aignatov-bio) [inline app/javascript/packs/vue/repository_item_error_sidebar.js:5]: You already defined inside component

File: app/javascript/vue/repository_item_sidebar/RepositoryItemErrorSidebar.vue [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<template>
+  <transition enter-from-class="translate-x-full w-0"
+              enter-active-class="transition-all ease-sharp duration-[588ms]"
```
Comment (aignatov-bio) [inline app/javascript/vue/repository_item_sidebar/RepositoryItemErrorSidebar.vue:3]: why 588ms?

File: app/javascript/vue/repository_item_sidebar/RepositoryItemErrorSidebar.vue [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    <div ref="wrapper" v-if="isShowing"
+      class='items-sidebar-wrapper bg-white gap-2.5 self-stretch rounded-tl-4 rounded-bl-4 sn-shadow-menu-lg h-full w-[565px]'>
+
+      <div class="w-full h-full pl-6 bg-white flex flex-col">
+        <div class="sticky top-0 right-0 bg-white flex z-50 flex-col h-[78px] pt-6">
+          <div class="header flex w-full h-[30px] pr-6">
+            <i id="close-icon" @click="close"
+              class="sn-icon sn-icon-close ml-auto cursor-pointer my-auto mx-0"></i>
+          </div>
+          <div id="divider" class="w-500 bg-sn-light-grey flex items-center self-stretch h-px mt-6 mr-6"></div>
```
Comment (aignatov-bio) [inline app/javascript/vue/repository_item_sidebar/RepositoryItemErrorSidebar.vue:16]: w-500 not exist

**Review by aignatov-bio — CHANGES_REQUESTED** [review]
I think all css can be done cleaner, a lot not necessary classes

File: app/javascript/vue/repository_item_sidebar/RepositoryItemErrorSidebar.vue (hunk 2 above)
Comment (sboursen-scinote) [reply app/javascript/vue/repository_item_sidebar/RepositoryItemErrorSidebar.vue:3]: That's the same duration as the items card

**Review by aignatov-bio — APPROVED** [review]

**Pull request merged**